jerry-core/jerry-port.h

```diff
@@ -139,6 +140,14 @@ bool jerry_port_get_time_zone (jerry_time_zone_t *tz_p);
  */
 double jerry_port_get_current_time (void);
 
+#ifndef CONFIG_DISABLE_ES2015_PROMISE_BUILTIN
+
+typedef uint32_t (*jerry_job_handler_t) (void *);
+
+void jerry_port_jobqueue_enqueue (jerry_job_handler_t handler, void *job_p);
```

Comment:
👍 Nice, this way any binding that needs to execute something asynchronously (for example `setTimeout()`) can use this to schedule the callback. In other words, the JobQueue feature is also useful for other (non-ES2015 Promise dependent) use cases.

Because of this, I think it makes sense to make a separate `CONFIG_DISABLE_ES2015_JOBQUEUE` feature flag.